gateway/api/v1/endpoint_decorator.py

```diff
@@ -0,0 +1,36 @@
+"""
+Endpoint decorator Module
+"""
+
+from functools import wraps
+from typing import Callable
+from api.v1.route_registry import RouteRegistry
+
+
+def endpoint(url_path: str, name: str = None):
```

Comment:
Should be this decorator outside `v1`?

Reply:
I was thinking on it, but, at the moment I dont know how we will manage different versions. So I think we should have a call to think on that.